translations/en/translation.json

```diff
@@ -20,7 +20,23 @@
       "type_TOO_MANY_ARGUMENTS": "{{func}}() was expecting no more than {{maxParams}} arguments, but received {{argCount}}. {{location}}",
       "type_WRONG_TYPE": "{{func}}() was expecting {{formatType}} for the {{position}} parameter, received {{argType}} instead. {{location}}"
     },
-    "location": "(at {{location}})",
+    "globalErrors": {
+      "reference": {
+        "notDefined": "There's an error due to \"{{symbol}}\" not being defined in the current scope {{location}}.\n\nIf you had defined it in your code, check its scope, spelling, and letter-casing (JavaScript is case-sensitive). For more:\n{{url1}}\n{{url2}}"
+      },
+      "stackSubseq": "▶️ Called from line {{line}} in \"{{func}}\" in {{file}} ({{location}})\n\n",
+      "stackTop": "▶️ Error at line {{line}} in \"{{func}}\" in {{file}} ({{location}})\n\n",
+      "syntax": {
+        "invalidToken": "There's a syntax error due to a symbol that JavaScript doesn't recognize or didn't expect at it's place.\nUsually this can when copying code from some other place. For example \" is supported but “ isn't.\nFor more: {{url}}",
```

Comment:
I don't know if the second sentence is necessary. It might too specific when there are so many characters that could cause this.

Reply:
This was supposed to be "can happen" :sweat_smile: 
The second sentence highlights the most common cause though yes I agree this might be troublesome if the cause is something else. Would the examples at the MDN link suffice?

Reply:
Oh yeah I carried over the lack of 'happen' 🤣 

I think the examples at the MDN link should suffice. Don't want to mislead people by being too specific even if it is the most common cause.